db/models.py

```diff
@@ -50,3 +60,50 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
+    pass
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(default=timezone.now, blank=True)
```

Comment:
The `created_at` field should use `auto_now_add=True` instead of `timezone.now` to automatically set the field to the current timestamp when the order is created. This avoids potential issues with timezone and ensures the field is set correctly.